package.json

```diff
@@ -66,6 +67,7 @@
     "@mongodb-js/compass-import-export": "^1.3.0",
     "@mongodb-js/compass-indexes": "0.0.10",
     "@mongodb-js/compass-instance": "^1.2.0",
+    "@mongodb-js/compass-instance-header": "^0.2.2",
```

Comment:
I think we should get rid of this, right? Since it is no longer being used. It should be removed from the renderer so that it's not required for it to run in standalone mode.

Reply:
Removed this plugin.